Int overflow on 32 bits systems

Golang 1.12.6 on i686 and armv7:
``` 
Testing    in: /builddir/build/BUILD/swag-0.19.0/_build/src
         PATH: /builddir/build/BUILD/swag-0.19.0/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/swag-0.19.0/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags "-X github.com/go-openapi/swag/version=0.19.0 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '"
      testing: github.com/go-openapi/swag
github.com/go-openapi/swag
FAIL	github.com/go-openapi/swag [build failed]BUILDSTDERR: # github.com/go-openapi/swag [github.com/go-openapi/swag.test]
BUILDSTDERR: ./convert_test.go:76:65: constant 9223372036854775807 overflows int
BUILDSTDERR: ./convert_test.go:92:65: constant 9223372036854775807 overflows int
BUILDSTDERR: ./convert_test.go:108:65: constant 9223372036854775807 overflows int
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -14,6 +14,12 @@ jobs:
     script:
     - go test -v
 
+  - go: 1.x
+    arch: arm
+    install: ~
+    script:
+    - go test -v
+
   # include linting job, but only for latest go version and amd64 arch
   - go: 1.x
     arch: amd64

--- a/.travis.yml
+++ b/.travis.yml
@@ -14,11 +14,11 @@ jobs:
     script:
     - go test -v
 
-  - go: 1.x
-    arch: arm
-    install: ~
-    script:
-    - go test -v
+  #- go: 1.x
+  #  arch: arm
+  #  install: ~
+  #  script:
+  #  - go test -v
 
   # include linting job, but only for latest go version and amd64 arch
   - go: 1.x

--- a/convert_test.go
+++ b/convert_test.go
@@ -73,7 +73,7 @@ func TestConvertFloat64(t *testing.T) {
 
 func TestConvertInt8(t *testing.T) {
 	validInts := []int8{0, 1, -1, math.MaxInt8, math.MinInt8}
-	invalidInts := []string{"1.233", "a", "false", strconv.Itoa(int(math.MaxInt64))}
+	invalidInts := []string{"1.233", "a", "false", strconv.FormatInt(int64(math.MaxInt64), 10)}
 
 	for _, f := range validInts {
 		c, err := ConvertInt8(FormatInt8(f))
@@ -89,7 +89,7 @@ func TestConvertInt8(t *testing.T) {
 
 func TestConvertInt16(t *testing.T) {
 	validInts := []int16{0, 1, -1, math.MaxInt8, math.MinInt8, math.MaxInt16, math.MinInt16}
-	invalidInts := []string{"1.233", "a", "false", strconv.Itoa(int(math.MaxInt64))}
+	invalidInts := []string{"1.233", "a", "false", strconv.FormatInt(int64(math.MaxInt64), 10)}
 
 	for _, f := range validInts {
 		c, err := ConvertInt16(FormatInt16(f))
@@ -105,7 +105,7 @@ func TestConvertInt16(t *testing.T) {
 
 func TestConvertInt32(t *testing.T) {
 	validInts := []int32{0, 1, -1, math.MaxInt8, math.MinInt8, math.MaxInt16, math.MinInt16, math.MinInt32, math.MaxInt32}
-	invalidInts := []string{"1.233", "a", "false", strconv.Itoa(int(math.MaxInt64))}
+	invalidInts := []string{"1.233", "a", "false", strconv.FormatInt(int64(math.MaxInt64), 10)}
 
 	for _, f := range validInts {
 		c, err := ConvertInt32(FormatInt32(f))

